src/components/vertical-divider/vertical-divider.spec.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import VerticalDivider from '.';
+import { StyledVerticalWrapper, StyledDivider } from './vertical-divider.style';
+import { assertStyleMatch, testStyledSystemSpacing } from '../../__spec_helper__/test-utils';
+import { baseTheme } from '../../style/themes';
+
+function render(props = {}) {
+  return mount(<VerticalDivider { ...props } />);
+}
+
+describe('VerticalDivider', () => {
+  describe('styles', () => {
+    testStyledSystemSpacing(
+      props => <VerticalDivider { ...props } />, {
+        pt: '24px', pb: '24px', pl: '24px', pr: '24px'
+      }
+    );
+
+    describe('with default values', () => {
+      it('matches expected', () => {
+        assertStyleMatch({
+          height: '100%',
+          borderLeft: `1px solid ${baseTheme.verticalDivider.color[80]}`,
+          display: 'inherit'
+        }, render().find(StyledDivider));
+      });
+    });
+
+    describe('with prop values set', () => {
+      it('matches expected when p prop is passed a value', () => {
```

Comment:
These tests aren't necessary, they're doing exactly what the `testStyledSystemSpacing` ones are doing - the whole point of that is that you don't have to write these yourself